packages/admin/admin/src/mui/menu/ItemGroup.tsx

```diff
@@ -1,31 +1,60 @@
-import { Box, ComponentsOverrides, Theme, Typography } from "@mui/material";
+import { Box, ComponentsOverrides, Theme, Tooltip, Typography } from "@mui/material";
 import { createStyles, WithStyles, withStyles } from "@mui/styles";
 import * as React from "react";
 
-export type MenuItemGroupClassKey = "root" | "title";
+export type MenuItemGroupClassKey = "root" | "title" | "titleContainer";
 
 const styles = (theme: Theme) =>
     createStyles<MenuItemGroupClassKey, MenuItemGroupProps>({
         root: { marginTop: theme.spacing(8) },
         title: {
             fontWeight: theme.typography.fontWeightBold,
-            fontSize: 14,
+            fontSize: ({ drawerOpen }) => (drawerOpen ? 14 : 12),
+            border: ({ drawerOpen }) => (drawerOpen ? `2px solid ${theme.palette.common.white}` : `2px solid ${theme.palette.grey[100]}`),
+            borderRadius: ({ drawerOpen }) => (drawerOpen ? "initial" : 20),
+            padding: ({ drawerOpen }) => (drawerOpen ? "0" : theme.spacing(2, 4)),
             lineHeight: "20px",
+            color: ({ drawerOpen }) => (drawerOpen ? `${theme.palette.common.black}` : `${theme.palette.grey[300]}`),
+        },
+        titleContainer: {
             borderBottom: `1px solid ${theme.palette.grey[50]}`,
-            padding: theme.spacing(2, 4),
+            display: "flex",
+            justifyContent: ({ drawerOpen }) => (drawerOpen ? "flex-start" : "center"),
+            padding: ({ drawerOpen }) => `${theme.spacing(2)} ${drawerOpen ? theme.spacing(4) : 0}`,
         },
     });
 
 export interface MenuItemGroupProps {
-    title?: React.ReactNode;
+    title: string;
```

Comment:
This is a breaking change and complicates I18N (FormattedMessage can't be used anymore). We need to find a different solution for this

Reply:
How is this a breaking change. The feature isn't shipped yet and gets merged as one from the feature branch🤔

Reply:
You're right, it isn't, sorry. Still, the I18N concern is still valid.

Reply:
True. I mean you could use translations but not the translation component... Changing it to `ReactNode` wouldn't work with the initials extraction though. We'd have to make it mandatory than to pass a `shortTitle` if the type of title is not a string.

Reply:
What do you suggest?

Reply:
How about something like this?

```tsx
function isFormattedMessage(
  node: React.ReactNode
): node is React.ReactElement<MessageDescriptor> {
  return React.isValidElement(node) && node.type === FormattedMessage;
}

function getInitials(title: React.ReactNode) {
  let titleAsString;

  if (typeof title === "string") {
    titleAsString = title;
  } else if (isFormattedMessage(title)) {
    titleAsString = intl.formatMessage(title);
  } else {
    throw new TypeError("Title must be either a string or a FormattedMessage");
  }

  const words = title.split(/\s+/).filter((word) => word.match(/[A-Za-z]/));
  return words.map((word) => word[0].toUpperCase()).join("");
}

```

Reply:
Good idea!